### PR TITLE
kernel options doc

### PR DESCRIPTION
**Pull request by minsa110**
For https://github.com/microsoft/vscode-jupyter/issues/12514

### CODE REVIEW

File: docs/datascience/jupyter-kernel-options.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+PageTitle: Jupyter Kernel Options in VS Code
+DateApproved: 11/21/2022
+MetaDescription: Descriptions, tips, and tricks on kernel selection options when working with Jupyter Notebooks in Visual Studio Code.
+MetaSocialImage: images/tutorial/social.png
+---
+
+# Jupyter Kernel Options in VS Code
+
+The VS Code notebooks’ kernel picker lists the following categories of connection methods to run the code in your Jupyter notebook:
+- Local Jupyter Kernels
```
Comment (rebornix) [inline docs/datascience/jupyter-kernel-options.md:15]: We now call them "Jupyter Kernels" in the extension.

Comment (minsa110) [reply docs/datascience/jupyter-kernel-options.md]: Fixed

File: docs/datascience/jupyter-kernel-options.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+## Python Environments
+
+The **Python Environments** category lists the Python environments that VS Code detects from the compute system it’s operating in (your desktop, Codespaces, remote server, etc.). It shows all Python environments grouped by type (e.g., conda, venv)—whether the [IPyKernel](https://ipython.readthedocs.io/en/stable/install/kernel_install.html) is installed or not.
+
+   > **Note**: You **__do not__** need to install [jupyter](https://pypi.org/project/jupyter/) into the Python environment you want to use. Only IPython and IPyKernel are required to launch a Python process as a kernel and execute code against your notebook, e.g., `pip install ipython ipykernel`. Visit our [wiki](https://github.com/microsoft/vscode-jupyter/wiki/Kernels-(Architecture)) to learn more.
```
Comment (DonJayamanne) [inline docs/datascience/jupyter-kernel-options.md:29]: Not sure why we need to mention `IPython`, 
The only requirement is for the user to have `IPyKernel` module to be installed, hence i would not mention `IPython`

File: docs/datascience/jupyter-kernel-options.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+## Existing Jupyter Server
+
+The **Existing Jupyter Server** category lists remote Jupyter servers previously connected. You can also use this option to connect to an existing Jupyter server running remotely or locally. Grab the URL for your Jupyter server, e.g., `http://<ip-address>:<port>/?token=<token>` and paste it in the `Enter the URL of the running Jupyter server` option to connect to the remote server and execute code against your notebook using that server.
+
+![Enter server URL](images/jupyter-kernel-options/select-enter-server-url.png)
+
+   > **Note**: When you’re starting the Jupyter server, ensure to: (1) allow all origins (e.g., `--NotebookApp.allow_origin = '*'`) to allow your servers to be accessed externally, and (2) set the notebook to listen to listen on all IPs (e.g., `--NotebookApp.ip = '0.0.0.0'`).
+
+Once connected, all active Jupyter sessions will appear on this list. You can create new session from the server’s kernelspec by selecting the `Notebook: Select Notebook Kernel` command -> `Select Another Kernel…` -> `Local Kernel Specs…` -> your server.
```
Comment (DonJayamanne) [inline docs/datascience/jupyter-kernel-options.md:39]: I dont think this is correct, the option should be
 `Notebook: Select Notebook Kernel` command -> `Select Another Kernel…` -> `Existing Jupyter Server…` -> your server.

@minsa110 you might want to try this at your end to ensure we're on the same page with this workflow

**Review by DonJayamanne — CHANGES_REQUESTED** [review]
Do not need to mention IPython
Workflow for remotes isn't correct, we don't need to mention `Local Kernel Specs`

File: docs/datascience/jupyter-kernel-options.md (hunk 2 above)
Comment (minsa110) [reply docs/datascience/jupyter-kernel-options.md:29]: Thanks, fixed :)

File: docs/datascience/jupyter-kernel-options.md (hunk 3 above)
Comment (minsa110) [reply docs/datascience/jupyter-kernel-options.md:39]: Confirmed and fixed :)

**Review by gregvanl — APPROVED** [review]

**Pull request merged**